### PR TITLE
More components

### PR DESCRIPTION
**Pull request by jdlrobson**
identifying more components as part of the epic task in #1092

There should be no css regressions. I am not styling these components any differently from before.

### CODE REVIEW

File: openlibrary/templates/lib/history.html [hunk 1]
```diff
@@ -33,7 +35,7 @@ <h2 class="collapse">
             $ rdf = page.key + ".rdf" + hist_addition
             $ json = page.key + ".json" + hist_addition
             $ opds = page.key + ".opds" + hist_addition
-            <span id="historyTools" class="smaller brown sansserif" style="float:right;padding-top:8px;">
+            <span id="historyTools" class="pageHistory__tools smaller brown sansserif" style="float:right;padding-top:8px;">
```
Comment (mekarpeles) [inline openlibrary/templates/lib/history.html]: Is there any way we can rm the inline styles here?

File: static/css/components/header-legacy.less [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        color: @black;
+        margin-right: 10px;
+        a,
+        span {
+          display: block;
+          width: 80px;
+          height: 22px;
+          padding: 10px 0 0 10px;
+          font-family: "Lucida Grande", Verdana, Geneva, Helvetica, Arial, sans-serif;
+          font-size: .6875em;
```
Comment (mekarpeles) [inline static/css/components/header-legacy.less]: How did we come up with this font-size? inspection of computed values?

File: static/css/components/page-history.less [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+.pageHistory__list {
+  font-size: .8em;
+}
+.pageHistory__tools {
+  font-size: .7em;
+}
+.pageHistory__list li {
+  &:last-child {
+    &:after {
+      content: '';
```
Comment (mekarpeles) [inline static/css/components/page-history.less]: why is empty `content` necessary here?

File: static/css/components/header-legacy.less (hunk 2 above)
Comment (jdlrobson) [reply static/css/components/header-legacy.less]: I have no idea :)
This change is just moving CSS I haven't had time to inspect it closely.
I've pulled this patch out into a new pull request:
https://github.com/internetarchive/openlibrary/pull/1331

**Comment by jdlrobson** [comment]
I'll look at the history page changes today.
In the mean time, I've pulled out the other changes into #1330, #1331, #1332

File: static/css/components/page-history.less (hunk 3 above)
Comment (jdlrobson) [reply static/css/components/page-history.less]: This needs an inline comment.

File: openlibrary/templates/lib/history.html (hunk 1 above)
Comment (jdlrobson) [reply openlibrary/templates/lib/history.html]: Done

File: openlibrary/templates/lib/history.html [hunk 4]
```diff
@@ -33,7 +35,7 @@ <h2 class="collapse">
             $ rdf = page.key + ".rdf" + hist_addition
             $ json = page.key + ".json" + hist_addition
             $ opds = page.key + ".opds" + hist_addition
-            <span id="historyTools" class="smaller brown sansserif" style="float:right;padding-top:8px;">
+            <span id="historyTools" class="pageHistory__tools smaller brown sansserif">
```
Comment (mekarpeles) [inline openlibrary/templates/lib/history.html]: thank you!

**Comment by mekarpeles** [comment]
lgtm!

**Pull request merged**